src/analyzer.ts

```diff
@@ -106,23 +107,20 @@ export class Analyzer {
    * see a mixture of pre-fork and post-fork contents when you analyze with a
    * forked analyzer.
    *
-   * Note: this feature is experimental.
+   * Note: this feature is experimental. It may be removed without being
+   *     considered a breaking change, so check for its existence before calling
+   *     it.
    */
   _fork(options?: ForkOptions): Analyzer {
-    const context =
-        options ? this._context._fork(undefined, options) : this._context;
+    const context = this._context._fork(undefined, options);
     return new Analyzer(context);
   }
```

Comment:
So here's my usage of `_fork` in an upcoming PR:

```
      analyzer:
          this._buildAnalyzer.analyzer
              ._fork({
                urlLoader: new FileMapUrlLoader(this.config.root, this.files)
              })
              .filesChanged(filesChanged),
```

Which is clearly going to result in a redundant analysis cache duplication, since I'm filesChanged()-ing it right after the fork.

It's minor, but I guess that's sort of a meh for me.  Do you think analyzer._fork should support a `filesChanged` option as well like `_fork({urlLoader: ..., filesChanged: [...]})`?